doc/content/examples/fuel_cycle/index.md

```diff
@@ -0,0 +1,111 @@
+\end{equation}
+
+\begin{equation}
+\label{eqn:t9} 
+\frac{dI_9}{dt} = (1-f_{8-11})\frac{I_8}{\tau_8} + \frac{I_{10}}{\tau_{10}} + \eta_2 \frac{I_2}{\tau_2} + \eta_6 \frac{I_6}{\tau_6} - \frac{I_9}{\tau_9} - I_9\lambda 
+\end{equation}
+\begin{equation}
+\label{eqn:t10} 
+\frac{dI_{10}}{dt} = f_{5-10}\frac{I_5}{\tau_5} + f_{9-10}\frac{I_9}{\tau_9} - \frac{I_{10}}{\tau_{10}} - I_{10}\lambda 
+\end{equation}
+\begin{equation}
+\label{eqn:t11} 
+\frac{dI_{11}}{dt} = f_{8-11}\frac{I_8}{\tau_8} + (1-f_{9-10})\frac{I_9}{\tau_9} - \frac{\dot{N}^{-}}{\eta_f f_b} - I_{11}\lambda 
+\end{equation}
+
+We instantiate the variables in the typical [`Variables`](/syntax/Variables) block, making sure to set the [!param](/Variables/family) attribute to `SCALAR` for each variable.
+The default initial condition is zero.
+
+!listing test/tests/fuel-cycle/fuel_cycle.i link=false block=Variables
+
+Next we initiate the [`ScalarKernels`](/syntax/ScalarKernels) block. We can model the time-dependent terms with [`ODETimeDerivative`](/syntax/ScalarKernels/ODETimeDerivative) objects
+and the other terms can be lumped in [`ParsedODEKernel`](/syntax/ScalarKernels/ParsedODEKernel) objects. We should have one [`ODETimeDerivative`](/syntax/ScalarKernels/ODETimeDerivative)
+ and one (or more) [`ParsedODEKernel`](/syntax/ScalarKernels/ParsedODEKernel) object(s) per equation above. Internally, TMAP8 will sum the contributions of each object, so we need to
+negate the [`ParsedODEKernel`](/syntax/ScalarKernels/ParsedODEKernel) equation from its representation above (move it to the left hand side). We use [`Postprocessors`](/syntax/Postprocessors) 
+to re-use the recurring variables
+
+!listing test/tests/fuel-cycle/fuel_cycle.i link=false block=ScalarKernels
+
+Finally, we define the [`Postprocessors`](/syntax/Postprocessors) and set their values to those referenced in [!cite](Abdou2021). Because the [`Postprocessors`](/syntax/Postprocessors) are
+inputs, not outputs, we must be careful to properly set the [!param](/Postprocessors/ConstantPostprocessor/execute_on) parameter. We also gather the values of the different variables in
+ separate [`Postprocessors`](/syntax/Postprocessors) and take their sum to obtain the total tritium inventory.
+
+!listing test/tests/fuel-cycle/fuel_cycle.i link=false block=Postprocessors
+
+
+Finally, we can set the [`Executioner`](/syntax/Executioner) block. In this case, we ask for a simple [`Transient`](/source/executioners/Transient.html) executioner with an exponentially growing 
+[`TimeStepper`](/syntax/Executioner/TimeStepper), [`IterationAdaptiveDT`](/source/timesteppers/IterationAdaptiveDT.html). 
+
+!listing test/tests/fuel-cycle/fuel_cycle.i link=false block=Executioner
+
+With these blocks set, we can now run the simulation and track tritium inventory. Implementing a high-fidelity model to replace the default values of specific postprocessors or to more accurately model 
+specific components is an exercise left for a later date.
```

Comment:
I got reasonable results, but have not yet made a concrete comparison to the paper.

Reply:
Can you make a new issue noting the need to add a results image and comparison to Abdou's paper, since you mentioned you had it on your list anyway? Would like to make sure it doesn't fall off the radar.